x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-ebs-overview.json

```diff
@@ -30,7 +30,7 @@
             "panelIndex": "1",
             "panelRefName": "panel_0",
             "title": "Volume Write Ops",
-            "version": "7.3.0"
+            "version": "7.4.0"
```

Comment:
Interesting that these versions work in current CI.

Reply:
yeah but the visualization version is tricky hmm